docs/docs/reference/metaprogramming/macros.md

```diff
@@ -602,7 +566,6 @@ sum
 ```
 
 Finally cleanups and dead code elimination:
-
```

Comment:
In this file, the blank lines before and after code blocks are removed in some cases. Is this by accident?

Reply:
From my experience it doesn't make any difference with top-level paragraphs, but **it does** with indented text (eg. list items). I'll revert the change.